pkg/ovs/openflow/logs.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Antrea Authors
```

Comment:
```suggestion
// Copyright 2020 Antrea Authors
```

Reply:
Thank you for your comment, I'll do as you suggested.